### PR TITLE
[4.0] Sprint - Remove sqlsrv comments

### PR DESCRIPTION
**Pull request by csthomas**
### Summary of Changes

Overview of sql queries marked as "Sqlsrv changes".

1. Remove comments.
2. Create text with 'CASE WHEN ..' in more simple way, for models create private dedicated method for that.
3. `quoteName` for table names
4. Replace `stdClass` to `\stdClass` in Category View library.

### Testing Instructions
Test search plugins
Test contacts: contact, category, featured view
Test content archived view

### Expected result
Should work as before.

### Documentation Changes Required
None.

### CODE REVIEW

**Comment by wilsonge** [comment]
Nice stuff :)

**Comment by dgt41** [comment]
CS failures?

File: components/com_contact/models/category.php [hunk 1]
```diff
@@ -406,6 +390,25 @@ public function &getChildren()
 	}
 
 	/**
+	* Generate column expression for slug or catslug.
+	*
+	* @param   JDatabaseQuery  $query  Current query instance.
+	* @param   string          $id     Column id name.
+	* @param   string          $alias  Column alias name.
+	*
+	* @return  string
```
Comment (zero-24) [inline components/com_contact/models/category.php]: Since version is missing :(

File: components/com_contact/models/contact.php [hunk 2]
```diff
@@ -370,6 +340,25 @@ protected function buildContactExtendedData($contact)
 	}
 
 	/**
+	* Generate column expression for slug or catslug.
+	*
+	* @param   JDatabaseQuery  $query  Current query instance.
+	* @param   string          $id     Column id name.
+	* @param   string          $alias  Column alias name.
+	*
+	* @return  string
```
Comment (zero-24) [inline components/com_contact/models/contact.php]: same here sicne tag missing :(

**Comment by csthomas** [comment]
I will create a PR and fix that

**Pull request merged**